docs/getting-started.md

```diff
@@ -39,7 +42,7 @@ title: Getting Started
       display: block;
       border-bottom-color: #05A5D1;
       border-radius: 3px;
-      margin: 2px 0px 2px 0px;
+      margin: 2px 0px 2px 0px !important;
```

Comment:
this fixes toggle alignment on mobile, it seems that `toggler` is also a class that is defined in docusaurus and that style overrides this one